pleskdistup/main.py

```diff
@@ -291,6 +292,30 @@ def read_resume_data(resume_path: PathType) -> ResumeData:
         raise
 
 
+@contextmanager
+def obtain_lock(lock_file: PathType) -> typing.Generator[bool, None, None]:
```

Comment:
"Obtain" is vague and close to "acquire" in its meaning. Propose to rename to "try_lock".